Wikipedia/Code/WMFLanguageCell.h

```diff
@@ -1,12 +1,14 @@
+#import "WMFTableViewCellWithCustomMinusButton.h"
 
 /// Table cell designed for displaying a language autonym & a page title in that language.
-@interface WMFLanguageCell : UITableViewCell
+@interface WMFLanguageCell : WMFTableViewCellWithCustomMinusButton
```

Comment:
@montehurd no need to worry about that… ALL cells support custom delete buttons on the right. No custom class is needed for that, so there shoudn't be much confusion.

And you can comment the header to explain the exact delete button that is being customized.

Also I forgot to mention, you the last part of the class name should be the type, the "adjectives" sound come first, so:

`WMFCustomDeleteButtonTableViewCell`

A good test is that classes generally shouldn't have the word "with" in them (unlike methods).